packages/wallet/src/cip30.ts

```diff
@@ -105,9 +99,8 @@ export const createWalletApi = (
   getNetworkId: async (): Promise<Cardano.NetworkId> => {
     logger.debug('getting networkId');
     const wallet = await walletReady;
-    const networkInfo = await firstValueFrom(wallet.networkInfo$);
-    // TODO: use networkInfo.network.id instead of magicToNetworkId
-    return magicToNetworkId(networkInfo.network.magic);
```

Comment:
Following the TODO comment, `genesisParameters.networkId` is used directly, removed obsolete `magicToNetworkId` mapping